### ISSUE TITLE
Accessibility Issues: Log In

### ISSUE THREAD

**Issue by aoik109** [lokeshsandhu/Surrey-Food-Bank-Team-c#48]
In the Log in page, there are 2 failed instances in the automated checks from "Accessibility Insights" involving:
- Color contrast issues
- Alternative text in image elements

Tab stops check has 2 failed instances:
- Keyboard navigation
- Focus indicator
both do not work with the "Don't have an account?" link and the "Forgot Password?" link.

<img width="533" height="567" alt="Image" src="https://github.com/user-attachments/assets/a067dc16-145b-44ff-890a-501bc585a38a" />

Full Report:
[2026-03-12 Login Page2 - FastPass.html](https://github.com/user-attachments/files/25961650/2026-03-12.Login.Page2.-.FastPass.html)
